src/appendices/spring/thymeleaf.rst

```diff
@@ -0,0 +1,108 @@
+.. _thymeleaf-commands:
+
+Thymeleaf Commands
+==================
+
+Thymeleaf is a template engine and Thymeleaf templates make up the views of our MVC application.
+In order to create our templates, we use attributes to pass data through controllers to our views out of the model.
```

Comment:
How about a slight rephrasing of this sentence to make the flow consistent with the MVC figure?  Maybe something like, "In order to create our templates, we use *attributes* to pass data from the model, through the controllers, and to our views."

Reply:
Done! Thanks!